primitives/vrf/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "vrf-primitives"
```

Comment:
I think that this crate is more about session/author keys, it's not VRF specific. So a more logical name could be `author-keys-primitives` or `session-keys-primitives`

Reply:
renamed to `session-keys-primitives`